lib/systems/src/PedalsSystem.cpp

```diff
@@ -78,17 +78,19 @@ bool PedalsSystem::_evaluate_min_max_pedal_implausibilities(int pedal_data, int
     }
     // FSAE EV.5.5
     // FSAE T.4.2.10
-    float float_pedal_data = float(pedal_data);
+    float float_pedal_data = static_cast<float>(pedal_data);
     float min_float = static_cast<float>(min);
     float max_float = static_cast<float>(max);
-    bool pedal_less_than_min = pedal_swapped ? (float_pedal_data > (min_float+pedal_margin)) : (float_pedal_data < (max_float-pedal_margin));
-    bool pedal_greater_than_max = pedal_swapped ? (float_pedal_data < (max_float-pedal_margin)) : (float_pedal_data > (min_float+pedal_margin));
+    bool pedal_less_than_min = pedal_swapped ? (float_pedal_data > (min_float+pedal_margin)) : (float_pedal_data < (min_float+pedal_margin)); //check the logic and fix
```

Comment:
What is the "check the logic and fix" comment? Does it need to be checked still or has it been checked and fixed